.github/workflows/site.yml

```diff
@@ -3,86 +3,91 @@ name: Docs and Site
-        # fsevents is not used on linux images, so use --no-optional (these plugins are suspect)
-        npm install --no-optional --save react react-copy-to-clipboard react-dom react-ga classnames carbon @carbon/addons-website carbon-components carbon-components-react carbon-addons-cloud carbon-icons gatsby gatsby-theme-carbon-starter markdown-it gatsby-plugin-manifest gatsby-plugin-slug gatsby-plugin-sitemap gatsby-plugin-sharp
+            echo "Check on Path"
+            pwd
+            cd docs/
+            npm install -g gatsby-cli
+            gatsby telemetry --disable
+            
+            # Install the packages
+            # fsevents is not used on linux images, so use --no-optional (these plugins are suspect)
+            npm install --no-optional --save react react-copy-to-clipboard react-dom react-ga classnames carbon @carbon/addons-website carbon-components carbon-components-react carbon-addons-cloud carbon-icons gatsby gatsby-theme-carbon-starter markdown-it gatsby-plugin-manifest gatsby-plugin-slug gatsby-plugin-sitemap gatsby-plugin-sharp
 
-        # make sure yarn is run to avoid this issue with #98123 WEBPACK
-        # add yarn --verbose to trace issues with yarn
-        
-        # don't use gatsby build directly per https://github.com/gatsbyjs/gatsby/issues/17557
-        # use the suggestion as a workaround due to path/issues in the gatsby component dependency of fsevents 
-        npm --prefix-paths run build 
-        cp -R public/ ../../public/
+            # make sure yarn is run to avoid this issue with #98123 WEBPACK
+            # add yarn --verbose to trace issues with yarn
+            
+            # don't use gatsby build directly per https://github.com/gatsbyjs/gatsby/issues/17557
+            # use the suggestion as a workaround due to path/issues in the gatsby component dependency of fsevents 
+            npm --prefix-paths run build 
+            cp -R public/ ../../public/
     - name: Grab the GH Pages Branch
       uses: actions/checkout@v1
       with: 
-        working-directory: gh-pages
-        ref: refs/heads/gh-pages
-        fetch-depth: 1
-        path: docs
-        token: ${{ secrets.GITHUB_TOKEN }} 
+            working-directory: gh-pages
+            ref: refs/heads/gh-pages
+            fetch-depth: 1
+            path: docs
+            token: ${{ secrets.GITHUB_TOKEN }} 
     - name: Commit and Add GH Pages
+      env: 
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            GITHUB_REPOSITORY: ${{ secrets.GITHUB_REPOSITORY }}
+            GITHUB_ACTOR: ${{ secrets.GITHUB_ACTOR }}
+            CI: true
       run: |
-        echo "cleaning up the prior files on the branch" 
-        if [ ! -f .nojekyll ]
-        then 
-          touch .nojekyll
-          rm -f _config.yml
-        fi
+            echo "cleaning up the prior files on the branch" 
+            if [ ! -f .nojekyll ]
+            then 
+            touch .nojekyll
+            rm -f _config.yml
```

Comment:
maybe indent this to make it clear its within the then?

Reply:
done